NSTTutorials/src/test/java/com/nst/tutorials/rest/runtimearguments/README.md

```diff
@@ -0,0 +1,73 @@
+[_Return to Tutorials Overview_](https://github.com/eBay/NSTSuite/tree/main/NSTTutorials)
+
+# Runtime Arguments
+
+### Overview
+
+NST has several runtime arguments that are supported by default. In this section we will review these runtime arguments, how to add your own custom runtime arguments, and how to add your own custom logger / format writer files.
+
+To see a list of the NST runtime arguments supported out of the box, please see [here](../../../../../../../../../NST/README.md#Runtime Arguments).
+
+### Topics Covered
+
+1. [Adding custom runtime arguments](#Adding custom runtime arguments)
+2. [Adding a custom logger](#Adding a custom logger)
+3. [Overriding an existing runtime argument](#Overriding an existing runtime argument)
+
+### References
+- [RuntimeConfigManager](../../../../../../../../../NST/src/main/java/com/ebay/runtime/RuntimeConfigManager.java)
+
+## Adding custom runtime arguments
+
+To add a custom runtime argument that can be read during a test (for example), you can leverage the `RuntimeConfigManager` . 
+
+To create the runtime argument object, create a class that implements `RuntimeConfigValue`. All the runtime arguments above implement this interface, which you can reference as an example.
+
+[Here](RuntimeArgumentsCustomExample.java) is one such example.
+The generic defined with the `RuntimeConfigValue` interface defines what value that runtime argument should return.
+
+Once you have created your runtime argument class, use `RuntimeConfigManager.*getInstance*().addRuntimeArgument` to add it to the manager. Then, you can reference the value that is set during runtime by referencing the key, as so:
+
+`RuntimeConfigManager.*getInstance*().getRuntimeArgumentValue(CustomRuntimeArgument.***KEY***);`
+
+See the [associated test class](RuntimeArgumentsTest.java) for an example of adding a custom runtime argument.
+
+## Overriding an existing runtime argument
+
```

Comment:
There is a difference between overriding existing arguments and overriding custom arguments. Please discuss both cases.